### PR TITLE
Request post

### PR DESCRIPTION
**Pull request by Tristan-Holtz**
# What does this PR do?

Conditionally renders the corresponding inputs depending on the category selected

# Where should your reviewer start?

RequestScreen.js

# What are the related issues?

This takes care of half of 24

# Anything else your reviewer should know?

### CODE REVIEW

File: screens/ItemDetailsScreen.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import * as WebBrowser from "expo-web-browser";
-import * as React from "react";
-import { StyleSheet, Text, View, TouchableOpacity } from "react-native";
-import { ScrollView } from "react-native-gesture-handler";
-import Colors from "../constants/Colors";
+import * as WebBrowser from 'expo-web-browser';
+import * as React from 'react';
+import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import Colors from '../constants/Colors';
```
Comment (cgeglio) [inline screens/ItemDetailsScreen.js:5]: my prettier changes these back to double quotes 🧐

Comment (Tristan-Holtz) [reply screens/ItemDetailsScreen.js:5]: I think mine is set to double quotes, I can switch it to singles in the settings so we don't just go back and forth haha

File: screens/RequestScreen.js [hunk 2]
```diff
@@ -1,21 +1,55 @@
...
+    if (!event.target.name) {
+      event.target.name = 'category';
+    }
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
   render() {
+    const inputComponent = (
+      <TextInput
+        placeholder={this.state.category === 'food' ? 'How many?' : 'How long?'}
```
Comment (cgeglio) [inline screens/RequestScreen.js:37]: Glad ya'll got the food issue figured out

File: screens/RequestScreen.js [hunk 3]
```diff
@@ -70,30 +137,31 @@ export class RequestScreen extends Component {
...
-    color: "black",
+    fontWeight: 'bold',
+    color: 'black',
     lineHeight: 35,
     margin: 10,
     marginTop: 20,
-    textAlign: "center"
-  }
+    textAlign: 'center',
+  },
```
Comment (cgeglio) [inline screens/RequestScreen.js:166]: oh boy, we're gonna have to figure out if we want single or double and configure prettier accordingly.

Comment (Tristan-Holtz) [reply screens/RequestScreen.js:166]: I'll switch mine lol

File: screens/ItemDetailsScreen.js (hunk 1 above)
Comment (cgeglio) [reply screens/ItemDetailsScreen.js:5]: sorry, i mean mine is set to change these all back to doubles, but it looks like this pr changes them all back to singles (so i think your prettier is set to singles?)

**Pull request merged**